src/ipynb/spark_ml.ipynb

```diff
@@ -34,7 +34,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    # Get the name of active Dataproc Interactive Session
</pre><p>nit: Get the name of the active Dataproc Serverless session</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>